release/readme.md

```diff
@@ -5,16 +5,31 @@
 * run `make configure` to regenerate `./configure` [checked by github actions]
 * update copyright headers
 * run `make tests`, `opam-rt` [checked by github actions & appveyor]
-* update the CHANGE file: take `master_changes.md` content to fil it
+* update the CHANGE file: take `master_changes.md` content to fill it
 
 ## Github release
 
 [ once bump version & changes PRs merged ]
-* tag the release (git tag -a 2.2.0; git push origin 2.2.0)
+* tag the release (git tag -am 2.2.0 2.2.0; git push origin 2.2.0)
 * /!\ Once the tag pushed, it can be updated [different commit] only in case of severe issue
 * create a release (or prerelease if intermediate release) draft on github based on your tag (https://github.com/ocaml/opam/releases/new)
-* generate opam artifacts, using `shell/release.sh`, it requires to have Docker install with several remotes, the different arches
-* add releases notes (content of `master_changes.md`) in the release
+* Install the github-unix package from opam (needed to get git-upload-release)
+* Create a new token at https://github.com/settings/tokens with the following rights: repo, write:packages, read:user, user:email
+* Create a new file at ~/.github/jar/infra with:
+```
+{
+  "scopes":[],
+  "token":"ghp_XXX",
+  "app":{"name":"infra","url":"https://developer.github.com/v3/oauth_authorizations/"},
+  "url":"https://api.github.com/authorizations/YYY",
+  "id":"YYY",
+  "note":"infra"
+}
+```
+  * Replace XXX by the token
+  * Replace YYY by its ID (you can find it in the url of the token page)
```

Comment:
I think we should probably replace this section with just "upload everything by hand", it’s faster and I still had issues yesterday with git-upload-release somehow

Reply:
Yeah in practice I have the same experience with unreliable `git-upload-release`, and uploading by hand is OK